core/telemetry/impl/service_impl.cpp

```diff
@@ -489,4 +482,41 @@ namespace kagome::telemetry {
   bool TelemetryServiceImpl::isEnabled() const {
     return enabled_;
   }
+
+  TelemetryServiceImpl::Bandwidth TelemetryServiceImpl::getBandwidth() {
+    static uint64_t previousBytesRead = 0;
+    static uint64_t previousBytesWritten = 0;
+
+    static auto previousTimeSent = std::chrono::high_resolution_clock::now();
```

Comment:
Make them telemetry members, or move to subobject (`struct Libp2pBandwidth { ...state... }`)
```c++
class TelemetryServiceImpl {
  using Clock = ...;
  uint64_t previousBytesRead, previousBytesWritten;
  std::optional<Clock::time_point> previousTimeSent;
}
```